### PR TITLE
Hide disabled option altogether

### PR DESCRIPTION
**Pull request by denis-yuen**
* would prefer tooltip, but for  https://github.com/angular/material2/issues/7953

### CODE REVIEW

File: src/app/search/map-friendly-values.pipe.ts [hunk 1]
```diff
@@ -35,7 +35,11 @@ export class MapFriendlyValuesPipe implements PipeTransform {
     ])],
     ['source_control_provider.keyword', new Map([
       ['GITHUB', 'github.com'], ['BITBUCKET', 'bitbucket.org'], ['GITLAB', 'gitlab.com'], ['DOCKSTORE', 'dockstore.org']
-    ])]
+    ])],
+    ['descriptor_tooltip', new Map([
+      ['CWL', 'Common Workflow Language'], ['WDL', 'Workflow Description Language'],
+      ['Nextflow', 'Nextflow coming soon!']
```
Comment (garyluu) [inline src/app/search/map-friendly-values.pipe.ts]: 'Nextflow' => 'NFL'

Comment (denis-yuen) [reply src/app/search/map-friendly-values.pipe.ts]: These tooltips should actually provide more lengthy information.

Comment (denis-yuen) [reply src/app/search/map-friendly-values.pipe.ts]: (FYI, discuss in person, I probably need to address the refresh issues)

**Comment by denis-yuen** [comment]
@garyluu is there anything to follow-up on for this?
@agduncan94 probably re-review

**Comment by garyluu** [comment]
don't think so

**Pull request merged**